### ISSUE TITLE
Uncaught SyntaxError: Unexpected token _ in JSON at JSON.parse

### ISSUE THREAD

**Issue by NekoSensei06** [nicoespeon/sass-graph-viz#17]
Hello,

i'm trying to use Sass-graph-viz, as a command line, but i get the same error of "Unexpected token _ in JSON" on most of my files; I tried to use it on a smaller file, with two imports :
`
@import '../base/variables';
@import '../base/mixins';
`

it works if the imports are on the same folder. It's a big project, so I need to have folders.

Here is the Json of that smaller file :
`[{"id":"programmation-nationale","label":"programmation-nationale","color":{"background":"#CCC","border":"#BBB","highlight":{"background":"#CCC","border":"#BBB"}}},{"id":"..\\base\\_variables","label":"..\\base\\_variables","color":{"background":"#9edb9b","border":"#57cc45","highlight":{"background":"#9edb9b","border":"#57cc45"}}},{"id":"..\\base\\_mixins","label":"..\\base\\_mixins","color":{"background":"#9edb9b","border":"#57cc45","highlight":{"background":"#9edb9b","border":"#57cc45"}}}]`

**Comment by nicoespeon** [comment]
Hi @AurelienJean 👋 

I'm not sure to understand your situation: what file are you trying to analyze? Is it a `something.scss` file that has the 2 imports inside? 

For example, I tried to run the library on the following structure:

```
📂base/
  |
  |__ variables.scss
  |__ mixins.scss
📂 src/
  |__ index.scss
```

With `index.scss` being:

```scss
@import '../base/variables';
@import '../base/mixins';
```

I ran `npx sass-graph-viz .` from the `src/` folder, and I got the following graph: 

![image](https://user-images.githubusercontent.com/1094774/80669750-28e21a00-8a73-11ea-8718-e12e71ec3e9b.png)

Could you give me a repro of the error?

**Comment by NekoSensei06** [comment]
hi,

I tried the same simple example, on a new folder; nothing in the files, expect the import in index.scss :
![image](https://user-images.githubusercontent.com/11072451/80813906-34485900-8bcb-11ea-94bd-98dd08c011a6.png)

I ran npx sass-graph-viz . from src /
No error on the npx command, but I get a blank page on chrome and an error in the chrome console : 
![image](https://user-images.githubusercontent.com/11072451/80815864-ae2e1180-8bce-11ea-8c39-4ae5e77dd394.png)

Firefox : 
![image](https://user-images.githubusercontent.com/11072451/80815355-c81b2480-8bcd-11ea-9148-11dd81ad97b5.png)


Here is the line 21 from index :
```html
</style><link href="https://fonts.googleapis.com/css?family=Orbitron" rel="stylesheet" defer><link href="preloader.css" rel="stylesheet"><link href="vis.4.21.0.min.css" rel="stylesheet" defer><script>var nodes = JSON.parse('[{"id":"index","label":"index","color":{"background":"#CCC","border":"#BBB","highlight":{"background":"#CCC","border":"#BBB"}}},{"id":"..\\base\\variables","label":"..\\base\\variables","color":{"background":"#CCC","border":"#BBB","highlight":{"background":"#CCC","border":"#BBB"}}},{"id":"..\\base\\mixins","label":"..\\base\\mixins","color":{"background":"#CCC","border":"#BBB","highlight":{"background":"#CCC","border":"#BBB"}}}]');
```
![image](https://user-images.githubusercontent.com/11072451/80815687-68714900-8bce-11ea-808b-6879795bda6e.png)


when I remove the import, or if I put the files on the same folder, it works : 
![image](https://user-images.githubusercontent.com/11072451/80813745-e895af80-8bca-11ea-9d7a-6e7d5e8c0361.png)

I'm on Windows 10, Vs Code; I ran npx on cmd, tried powershell and git bash.

**Comment by NekoSensei06** [comment]
I tried it on a debian WSL, and it works; so it seems like a Windows problem.

So, at least I can use it that way :)

**Comment by nicoespeon** [comment]
Thanks for the details @AurelienJean, that was helpful.

So yeah, I know what the problem is. And yep, it's Windows-specific.

On Windows, paths use the backslash symbol instead of the regular slash. Thus, we end up with the following string: 

```
'[{"id":"index","label":"index","color":{"background":"#CCC","border":"#BBB","highlight":{"background":"#CCC","border":"#BBB"}}},{"id":"..\\base\\variables","label":"..\\base\\variables","color":{"background":"#CCC","border":"#BBB","highlight":{"background":"#CCC","border":"#BBB"}}},{"id":"..\\base\\mixins","label":"..\\base\\mixins","color":{"background":"#CCC","border":"#BBB","highlight":{"background":"#CCC","border":"#BBB"}}}]'
```

This can't be passed into `JSON.parse()`. We need to escape the backslashes again, so we have `\\\\` instead of `\\`.

I'll see what I can do 👍

**Comment by nicoespeon** [comment]
@AurelienJean I released v2.3.1 with an attempt to fix this. I think it's OK but I don't have a Windows OS, so I can't confirm.

Can you tell me if it works now?

**Comment by NekoSensei06** [comment]
It's working !

Thanks a lot for the fix and your product, it will help me optimize my scss architecture :)
